src/hexo/themes/documentation/layout/partials/contributors.hbs

```diff
@@ -14,7 +14,9 @@
             <div class="layout layout--quarters">
             {{#each devs}}
                 <div class="module module--contributors">
-                    <img src="{{avatar}}?s=100" width="100" alt="Antón Molleda">
+                    <a {{#if website}} href="{{website}}"{{else}}href="#"{{/if}}>
+                        <img src="{{avatar}}?s=100" width="100" alt="Antón Molleda">
```

Comment:
That code was already in there, and it looked like a mistake to me. I'll go ahead and fix that too.